src/Raven.Client/ServerWide/Operations/Logs/SetLogsConfigurationOperation.cs

```diff
@@ -20,6 +20,18 @@ public class Parameters
             public TimeSpan? RetentionTime { get; set; }
             public Size? RetentionSize { get; set; }
             public bool Compress { get; set; }
+
+            public Parameters()
+            {
+            }
+            
+            public Parameters(GetLogsConfigurationResult currentLogsConfiguration)
```

Comment:
we added this constructor to have user live easier. When we want to modify single propertly and leave other with same values, we use GetLogsOperation which returns GetLogsConfigurationResult, then we can use this object to initialize parameters for set operation with current values.

Reply:
See usage in the new test